### PR TITLE
Ct 2590 deactivate old team populate moved to

### PR DESCRIPTION
**Pull request by danielatdigital**
## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
 
### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2590?atlOrigin=eyJpIjoiNjlmNGM0NzQ0NzEyNGJlZGJiYzQyMzJmNGY5OGQ4NjQiLCJwIjoiaiJ9

### CODE REVIEW

File: .rakeTasks [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
...s:templates:copy" taksId="copy" /></RakeGroup><RakeTask description="" fullCmd="rails:update" taksId="update" /><RakeGroup description="" fullCmd="" taksId="update"><RakeTask description="" fullCmd="rails:update:bin" taksId="bin" /><RakeTask description="" fullCmd="rails:update:configs" taksId="configs" /></RakeGroup></RakeGroup><RakeGroup description="" fullCmd="" taksId="railties"><RakeGroup description="" fullCmd="" taksId="install"><RakeTask description="" fullCmd="railties:install:migrations" taksId="migrations" /></RakeGroup></RakeGroup><RakeGroup description="" fullCmd="" taksId="semaphore"><RakeTask description="" fullCmd="semaphore:prepare_db_config" taksId="prepare_db_config" /></RakeGroup><RakeGroup description="" fullCmd="" taksId="sidekiq"><RakeTask description="" fullCmd="sidekiq:status" taksId="status" /></RakeGroup><RakeTask description="" fullCmd="tmp" taksId="tmp" /><RakeTask description="" fullCmd="tmp/cache" taksId="tmp/cache" /><RakeTask description="" fullCmd="tmp/cache/assets" taksId="tmp/cache/assets" /><RakeTask description="" fullCmd="tmp/pids" taksId="tmp/pids" /><RakeTask description="" fullCmd="tmp/sockets" taksId="tmp/sockets" /></RakeGroup></Settings>
```
Comment (Polsonby) [inline .rakeTasks]: Is this file part of this work? maybe remove if not...

File: .generators [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...eratorsGroup><Generator name="active_record:migration" /><Generator name="active_record:model" /><Generator name="active_record:observer" /><Generator name="active_record:session_migration" /><Generator name="controller" /><Generator name="erb:controller" /><Generator name="erb:mailer" /><Generator name="erb:scaffold" /><Generator name="generator" /><Generator name="helper" /><Generator name="integration_test" /><Generator name="mailer" /><Generator name="metal" /><Generator name="migration" /><Generator name="model" /><Generator name="model_subclass" /><Generator name="observer" /><Generator name="performance_test" /><Generator name="plugin" /><Generator name="resource" /><Generator name="scaffold" /><Generator name="scaffold_controller" /><Generator name="session_migration" /><Generator name="stylesheets" /><Generator name="test_unit:controller" /><Generator name="test_unit:helper" /><Generator name="test_unit:integration" /><Generator name="test_unit:mailer" /><Generator name="test_unit:model" /><Generator name="test_unit:observer" /><Generator name="test_unit:performance" /><Generator name="test_unit:plugin" /><Generator name="test_unit:scaffold" /></GeneratorsGroup></Settings>
```
Comment (Polsonby) [inline .generators]: Is this file part of this work? maybe remove if not...

File: .rakeTasks (hunk 1 above)
Comment (danielatdigital) [reply .rakeTasks]: Removed

File: .generators (hunk 2 above)
Comment (danielatdigital) [reply .generators]: Removed

**Review by Polsonby — APPROVED** [review]
Great stuff

**Pull request merged**